app/build.gradle

```diff
@@ -5,12 +5,12 @@ def keystore = new Properties()
 keystore.load(new FileInputStream(rootProject.file("keystore.properties")))
 
 android {
-    compileSdkVersion 33
+    compileSdkVersion 34
 
     defaultConfig {
         applicationId "io.rapidpro.androidchannel"
-        minSdkVersion 21
-        targetSdkVersion 33
+        minSdkVersion 23
```

Comment:
https://developer.android.com/reference/android/app/PendingIntent#FLAG_IMMUTABLE

used in 

https://github.com/rapidpro/android-channel/pull/31/files#diff-0316112d26162e8bc17d0417c6f16b845995e0b426b828b971b5c616ccb6a885R229